cmd/installer/cli/install.go

```diff
@@ -590,7 +588,7 @@ func preRunInstallKubernetes(_ *cobra.Command, flags *InstallCmdFlags, _ kuberne
 		}
 	}
 
-	restConfig, err := flags.kubernetesEnvSettings.RESTClientGetter().ToRESTConfig()
+	restConfig, err := clientcmd.BuildConfigFromFlags("", flags.kubernetesEnvSettings.KubeConfig)
```

Comment:
why change this? is it equivalent? what if i dont explicitly set the kubeconfig as a flag?

Reply:
good catch